src/lib/y2security/security_policies/manager.rb

```diff
@@ -106,12 +130,27 @@ def enabled_policy?(policy)
       #   only display rules that it can fix.
       # @return [Hash<Policy, Array<Rule>>]
       def failing_rules(config, scope: nil, include_disabled: false)
-        enabled_policies.each_with_object({}) do |policy, result|
+        @last_result = enabled_policies.each_with_object({}) do |policy, result|
```

Comment:
Do we need to catch this? Checking rules is quite fast and there is some chance to use outdated results.

Reply:
We are catching the results because we need them *at the end* of the installation. But if the TargetConfig remains the same, we can avoid it.